v3/setup/images-control.sh

```diff
@@ -6,11 +6,11 @@ if [ "${NODE_ROLE}" != "control" ]; then
     exit 0
 fi
 
-if [ "$(etcdctl get images-control-bootstrapped)" == "true" ]; then
+if [ "$(etcdctl get /bootstrap.service/images-control-bootstrapped)" == "true" ]; then
```

Comment:
The above is okay then, my comment was more that `/bootstrap.service` already implies that whatever follows (images-control-bootstrapped) is being bootstrapped. I.e., we don't call it `/bootstrap.service/flight-director-bootstrapped`